ci/configure_v5.sh

```diff
@@ -1,3 +1,9 @@
 #!/bin/bash -ex
 
 docker-compose exec -T conjur_5 conjurctl wait -r 30 -p 80
+
+api_key=$(docker-compose exec -T conjur_5 bash -c 'rails r "puts Role[%Q{cucumber:user:admin}].api_key" 2>/dev/null')
```

Comment:
What prompted this change? Seems unrelated to annotations.

Reply:
Yeah, it's more for consistency - the v4 configure loads the policy, so I like the idea of v5 configure doing the same. There were some other experiments I was doing that initially prompted me to put this in here, but I think it belongs here.